iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -37,15 +38,30 @@ class ConversationMessagesCell: UICollectionViewCell {
     private var state: ConversationUIState = .read
 
     /// The conversation containing all the messages.
-    var conversation: MessageSequence?
+    var conversation: Conversation? {
+        return self.conversationController?.conversation
+    }
+    private(set) var conversationController: ConversationController?
+    private var shouldShowLoadMore: Bool {
+        guard let conversationController = self.conversationController else {
+            return false
+        }
+
+        if conversationController.messages.count < .messagesPageSize {
+            return false
+        }
+        return !conversationController.hasLoadedAllPreviousMessages
```

Comment:
The conversation controller is annoying because "hasLoadedAllPreviousMessages" is always false at first, even if all the messages are loaded. It only flips to true once you call "loadMessage".

To get around this, I first check to see how many messages are loaded and return false if it's less than the default page size. There's an edge case where the load more cell will show if there are exactly 25 messages are in the conversation but it's probable not a huge deal.